File: Team1/RiyaSingh/XOR.java
class Solution {
    public int xorOperation(int n, int start) {
        int[] nums = new int[n];
        for(int i=0;i<n;i++){
            nums[i] = start + 2*i;
        }
        for(int i=1;i<n;i++){
            start = (start ^ nums[i]);
        }
        return start;
    }
}

File: Team1/Mansi Sharma/xor.cpp
class Solution {
public:
    int xorOperation(int n, int start) {
        int nums[n];
        for (int i = 0; i < n; i++) {
            nums[i] = start + 2*i;
        }
        int ans = 0;
        for (int i = 0; i < n; i++) {
            ans = ans xor nums[i];
        }
        return ans;
    }
};
